### DISCUSSION TITLE
Developer meeting (March 6th): The Crazyflie Supervisor

### DISCUSSION THREAD

**Discussion by ataffanel** [bitcraze/discussions#1196] [Announcements]
The next developer meeting will be on the 6th of March 2024, we will talk about the Crazyflie Supervisor subsystem. We have made some change to the supervisor recently, and we will continue working more on it in the next couple of weeks mostly preparing for the arrival of the Brushless Crazyflie. @tobbeanton will talk about the current state and what we are working on.

Date: Wednesday, March 6th, 2024
Time: 3 PM Central European Time (check your [timezone](https://www.worldtimebuddy.com/?qm=1&lid=2692969,5128581,5368361,1850147&h=2692969&date=2024-3-6&sln=15-16&hf=0))
Topic: Crazyflie Supervisor
Hosts: @tobbeanton 
Meeting link: Meeting is over, watch the recording here: https://youtu.be/VcW6Uw5nLz8

Agenda
15:00 - 15:30: Presentation, hands-on and discussion
15:30 - 16:00: Generic support questions

Indicate with 👍 if you are planning to join!

**Comment by knmcguire** [comment]
The recording can be found here https://youtu.be/VcW6Uw5nLz8
